content/ri/using-redisinsight/add-instance.md

```diff
@@ -27,23 +27,13 @@ Troubleshooting:
 
 ## Connecting to ElastiCache
 
-If you want to work with ElastiCache Redis instances with RedisInsight, you have two options:
+Connecting to AWS ElastiCache can be particularly problematic since ElastiCache caches cannot be accessed from outside the VPC, as they don't have public IP addresses assigned to them.
 
-1. [Install RedisInsight on an EC2 instance]({{< relref "/ri/installing/install-ec2.md" >}}) that has access to the ElastiCache Redis instance.
-1. Create a SSH tunnel through an EC2 instance that can access the ElastiCache Redis instance.
+If you want to work with ElastiCache Redis caches using RedisInsight, you have two options:
 
-### Using an SSH Tunnel
```

Comment:
why did we delete this level of detail.  Is it no longer relevant?

Reply:
I would really keep this

Reply:
I removed it because it is essentially a poor man's NAT. But it is definitely much easier to set up. Agreed that we should keep it.